python/ray/_private/utils.py

```diff
@@ -1599,6 +1600,87 @@ def split_address(address: str) -> Tuple[str, str]:
     return (module_string, inner_address)
 
 
+def _add_url_query_params(
+    url: str, params: Dict[str, str], override: bool = False
```

Comment:
we don't need `override` right? Can we simplify the code and introduce later if needed?